public/jsx/app.jsx

```diff
@@ -23,7 +24,10 @@ var routes = (
   <Route handler={App}>
     <Route name="login" path="/" handler={Login}/>
     <Route name="card-form" path="/payment/card/" handler={CardDetails}/>
-    <Route name="complete" path="/payment/complete/" handler={CompletePayment}/>
+    <Route
+      name="complete" path="/payment/complete/" handler={CompletePayment}/>
+    <Route
+      name="card-listing" path="/payment/card-list/" handler={CardListing}/>
```

Comment:
I think we should get in the habit of calling these payment methods (I've been going with ``paymethods`` for short). It doesn't have to be in this commit but maybe later. The reason is we'll likely have more than just credit cards at some point.